tests/unit/accounts/test_models.py

```diff
@@ -102,23 +102,20 @@ def test_query_by_email_when_not_primary(self, db_session):
 
     def test_recent_events(self, db_session):
         user = DBUserFactory.create()
-        recent_event = DBUserEventFactory(source=user, tag="foo", ip_address="0.0.0.0")
+        recent_event = DBUserEventFactory(source=user, tag="foo")
         legacy_event = DBUserEventFactory(
             source=user,
             tag="wu",
-            ip_address_string="0.0.0.0",
             time=datetime.datetime.now() - datetime.timedelta(days=1),
         )
         stale_event = DBUserEventFactory(
             source=user,
             tag="bar",
-            ip_address="0.0.0.0",
             time=datetime.datetime.now() - datetime.timedelta(days=91),
         )
 
         assert user.events.all() == [recent_event, legacy_event, stale_event]
         assert user.recent_events.all() == [recent_event, legacy_event]
-        assert user.recent_events.all()[-1].ip_address == "0.0.0.0"
```

Comment:
This assertion was removed since the `db_session` doesn't have the `request` object, and we're also not using the `record_event()` method that would make the IP address available.
We _could_ also remove/rename the concept of `legacy_event`, since there's no real difference between the two structurally.